mobile/android/app/build.gradle

```diff
@@ -55,6 +55,9 @@ android {
         targetSdkVersion flutter.targetSdkVersion
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
+        manifestPlaceholders += [
+                'appAuthRedirectScheme': 'app.alextran.immich'
+        ]
```

Comment:
Is this something specific to using OAuth?

Reply:
Yes it is needed for OAuth, because the provider needs a URI to redirect the user to. In this case OAuth is redirecting the user to the application itself. You can see here for the usage https://github.com/EnricoBilla/immich/blob/e059cf220c9de9f7709266b62ad917808ce390d6/mobile/lib/modules/login/providers/oauth2.provider.dart#L7